packages/cli/src/cmds/inventory/readIndexFile.ts

```diff
@@ -1,7 +1,13 @@
 import { readFile } from 'fs/promises';
 import { basename, dirname, join } from 'path';
+import { exists } from '../../utils';
 
-export default async function readIndexFile(appmap: string, indexFileName: string) {
+export default async function readIndexFile(
+  appmap: string,
+  indexFileName: string
+): Promise<any[] | any | undefined> {
```

Comment:
This would be more aptly `Promise<Record<string, unknown> | Array<Record<string, unknown>> | undefined`.

The usage of `any` will collapse the type definition down to `any`, which won't require the compiler to check for `undefined` or whether or not the return type is an object or an array.

Though, even `Record<string, unknown` is insufficient because these types should be known.